examples/riot-kitchen-sink/public/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <!--
+      Notice the use of %PUBLIC_URL% in the tag above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```

Comment:
Riot App?

Reply:
Is not the storybook app a react app anyway ? Event if riot is loaded inside ?

Reply:
This is implementation details, no reason to put them in page title

Reply:
If you want to run riot example outside the storybook, will it be a react app?

Reply:
Actually you can, it's not used